tests/utils/errors.test.ts

```diff
@@ -0,0 +1,8 @@
+import { AccessTokenError, HandlerError } from '../../src/utils/errors';
+
+describe('errors', () => {
+  test('should be instance of themselves', () => {
```

Comment:
Believe it or not, this test will fail in TS with `target: es5` without the `Object.setPrototypeOf(this, ThisError.prototype);` workaround